src/bean/grid.cljs

```diff
@@ -134,7 +110,7 @@
          (spill grid))))
 
 (defn parse-grid [grid]
-  (util/map-on-matrix content->cell grid))
+  (util/map-on-matrix value/from-cell grid))
```

Comment:
This name seems incorrect, it's more like `to-cell`. This is the input to the interpreter, the interpreter returns a `value`. See `ast-result`.